### PR TITLE
bootstrap org usage dashboard

### PR DESCRIPTION
**Pull request by xp-1000**
image

UPDATE: 

- new implementation using terraform 0.13 for_each on module
- support parent and child org
- support detectors links
- deploy 2 dashboards for parent, only one for child

See related PR https://github.com/claranet/terraform-signalfx-detectors/pull/74/files

### CODE REVIEW

File: organization/usage/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Orgnization usage dashboards
+
+![](././img/dashboard_child.png)
```
Comment (Shr3ps) [inline organization/usage/README.md:3]:
```suggestion
![](./img/dashboard_child.png)
```

File: organization/usage/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## Detectors links
+
+This is possible to add [detectors](https://github.com/claranet/terraform-signalfx-detectors/tree/master/organization/usage) links to their corresponding chart:
```
Comment (Shr3ps) [inline organization/usage/README.md:7]:
```suggestion
It is possible to add [detector](https://github.com/claranet/terraform-signalfx-detectors/tree/master/organization/usage) links to their corresponding chart:
```

File: organization/usage/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+locals {
+  container_ratio_detector = {
+    id   = tostring(module.signalfx-detectors-organization-usage.container_ratio_id[0]),
+    name = module.signalfx-detectors-organization-usage.container_ratio_name[0],
+  }
+}
+
+module "signalfx-dashboards-organization-usage" {
+  source = "github.com/claranet/terraform-signalfx-dashboards.git//organization/usage?ref=usage"
```
Comment (Shr3ps) [inline organization/usage/README.md:24]: ref= will be probably change before merge

File: organization/usage/dashboard-usage/variables.tf [hunk 4]
```diff
@@ -0,0 +1,52 @@
+variable "multiplier" {
+  description = "Multiplier should be 2 for \"enterprise\" or 1 for \"pro\" plan (i.e. 2 = 200 custom metrics and 20 containers per host)."
```
Comment (Shr3ps) [inline organization/usage/dashboard-usage/variables.tf:2]:
```suggestion
  description = <<EOD
Multiplier should be 2 for "enterprise" or 1 for "pro" plan.
(i.e. 2 = 200 custom metrics and 20 containers per host)
EOD
```

File: organization/usage/outputs.tf [hunk 5]
```diff
@@ -0,0 +1,15 @@
+output "dashboard_group_id" {
+  description = "The ID of the usage dashboard group"
+  value       = signalfx_dashboard_group.usage.id
+}
+
+#output "dashboard_child_id" {
```
Comment (Shr3ps) [inline organization/usage/outputs.tf:6]: you will drop those dead code before merge?

**Review by RomainSup35 — APPROVED** [review]

**Pull request merged**